tests/browser-automated-tests-playwright/index.spec.js

```diff
@@ -184,5 +184,7 @@ test.describe("Subscribie tests:", () => {
   // 1. Transacion filter by name and plan title
   // 2. 2.A pause, resume and 2.B cancel subscription test. 
   order_plan_with_subscription_and_upfront_charge = require('./tests/293_subscriber_order_plan_with_recurring_and_upfront_charge');
+
+  clearing_connect_account_id = require('./tests/clear_connect_account_id.js');
```

Comment:
Please change to 

`clear_stripe_connect_account_id`  (clear rather than clearing) , (stripe to reduce ambiguity)

@joeltejeda